presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -301,7 +301,7 @@ protected static ResultSet getColumns(JdbcTableHandle tableHandle, DatabaseMetaD
 
     protected Optional<ColumnMapping> getForcedMappingToVarchar(JdbcTypeHandle typeHandle)
     {
-        if (typeHandle.getJdbcTypeName().isPresent() && jdbcTypesMappedToVarchar.contains(typeHandle.getJdbcTypeName().get())) {
+        if (typeHandle.getJdbcTypeName().isPresent() && jdbcTypesMappedToVarchar.contains(typeHandle.getJdbcTypeName().get().toLowerCase(ENGLISH))) {
```

Comment:
`jdbcTypesMappedToVarchar` is protected, so it requires modications in subclasses.

instead, i'd consider using `TreeSet` with `java.lang.String#CASE_INSENSITIVE_ORDER` for `jdbcTypesMappedToVarchar`